app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/inspections/AppEngineDeprecatedRuntimeInspection.java

```diff
@@ -94,8 +94,10 @@ public String getFamilyName() {
 
     @Override
     public void applyFix(@NotNull Project project, @NotNull ProblemDescriptor descriptor) {
-      XmlTag xmlTag = (XmlTag) descriptor.getPsiElement();
-      xmlTag.getValue().setText(APP_ENGINE_JAVA8_RUNTIME_VALUE);
+      if (descriptor.getPsiElement() != null && descriptor.getPsiElement() instanceof XmlTag) {
```

Comment:
I wonder why this happened, my understanding was we registered this inspection to work with a specific file type? Anyway, good to know for the future inspections.

Reply:
exactly. that was the next step to try and determine how to reproduce to see if we need to do a patch release. If you could help with this that would be great